spec/basicConf.js

```diff
@@ -13,6 +13,8 @@ exports.config = {
 
   // Exclude patterns are relative to this directory.
   exclude: [
+    // TODO(cnishina): update when mouseMoveTo works in the next release of selenium-webdriver.
+    'basic/action_spec.js',
```

Comment:
Instead of doing this, could you change the test in action_spec.js to 'xit()' and put the comment there? I think it'll be easier to keep track of that way.